src/App.tsx

```diff
@@ -59,6 +60,12 @@ export default function () {
               <Route path="/wallet">
                 <BurnerWallet />
               </Route>
+              <Route path="/404">
```

Comment:
Let's do it like this:
![image](https://user-images.githubusercontent.com/49658988/194390754-3c1dbb35-343b-4d33-987f-f61b13b95fa3.png)

Reply:
I mean multiple `path` as one prop

Reply:
Meh, deprecated